File: neoforge/src/main/java/cn/breadnicecat/lovemod/neoforge/LoveModImpl.java
package cn.breadnicecat.lovemod.neoforge;

import cn.breadnicecat.lovemod.LoveMod;
import net.neoforged.fml.common.Mod;

@Mod(LoveMod.MOD_ID)
public class LoveModImpl {
	public LoveModImpl() {
		LoveMod.LOGGER.info("Loading Forge");
		LoveMod.init();
	}
}

File: neoforge/src/main/kotlin/cn/breadnicecat/lovemod/neoforge/LoveModImpl.kt
package cn.breadnicecat.lovemod.neoforge

import cn.breadnicecat.lovemod.LoveMod
import net.neoforged.fml.common.Mod

@Mod(LoveMod.MOD_ID)
class LoveModImpl {
    init {
        LoveMod.LOGGER.info("Loading Forge")
        LoveMod.init()
    }
}
